assets.go

```diff
@@ -165,31 +164,6 @@ func (l *Loader) Load(onFinish func()) {
 	onFinish()
 }
 
-type Image interface {
-	Data() interface{}
-	Width() int
-	Height() int
-}
-
-func LoadShader(vertSrc, fragSrc string) *gl.Program {
```

Comment:
See https://github.com/EngoEngine/engo/pull/247/files#r60846631 as well. 

This method should be able to run on the web? Since the `EngoEngine/gl` package has build stuff for netgo?